src/arc.js

```diff
@@ -492,7 +492,8 @@ ChartInternal.prototype.redrawArc = function (duration, durationForExit, withTra
         backgroundArc
             .exit()
             .remove();
-
+        
+        $$.innerRadius = $$.innerRadius || 0;
```

Comment:
I think this is not a right place to do this. How about checking this in `updateRadius`?